src/main/java/epam/gym/config/JPAConfig.java

```diff
@@ -0,0 +1,91 @@
+package epam.gym.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.dao.annotation.PersistenceExceptionTranslationPostProcessor;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+
+@Configuration
+@ComponentScan("epam.gym")
+@PropertySource("classpath:application.properties")
+@EnableTransactionManagement
```

Comment:
I believe we can remove these annotations for one of config classes not to dublicate
@ComponentScan("epam.gym")
@PropertySource("classpath:application.properties")
@EnableTransactionManagement